metabase/metabase
# Make ace work without inline styles

Related to https://github.com/metabase/metabase-private/issues/73

How to verify ace changes:
- New -> SQL query 
- The editor should be functional - without these fixes it doesn't allow you to input text

How to verify Metabase them changes:
- Admin -> Embedding -> Enable
- Question -> Share -> Embed in app -> View code
- Embedding code should be displayed and styled with our styles

<img width="1728" alt="Screenshot 2023-07-19 at 19 27 42" src="https://github.com/metabase/metabase/assets/8542534/1d0d70f8-82e5-4834-8b58-3d9cd78bf403">
